x-pack/plugins/cloud/public/types.ts

```diff
@@ -44,6 +44,10 @@ export interface CloudStart {
    * The full URL to the users and roles page on Elastic Cloud. Undefined if not running on Cloud.
    */
   usersAndRolesUrl?: string;
+  /**
+   * The full URL to the serverless projects page on Elastic Cloud. Undefined if not running in Serverless.
+   */
+  projectsUrl?: string;
```

Comment:
Not sure about this, because this could also go inside `serverless.*` 
The problem with putting it inside serverless is that it would be tricky to test this locally as `serveless.*` must specify `projectid`

Reply:
I think it is ok here. Serverless is Cloud. Now if we want to group the urls under a "serverless" parent that could be an option

```ts
interface CloudStart {
  serverless: {
    projectsUrl?: string;
  }
  ...
}

Reply:
there is already `serverless` parent config, but I am on a fence about putting it there because a valid servleless config requires `projectid` and the testing local config that just includes `serverless.prjectsUrl` would be considered invalid